`run_plugins_spec` test is leaking into other tests

`@ipc.send` in `run_plugins_spec` is somehow leading to failing tests later on:

https://github.com/cypress-io/cypress/blob/bbff24dc6799a2e9464eebff8e1f9ad468fa5635/packages/server/test/unit/plugins/child/run_plugins_spec.coffee#L67-L80

https://circleci.com/gh/cypress-io/cypress/123148#tests/containers/1 -
```
AssertionError: expected 'error' to equal 'load:error'
    at Object.ipc.send (/root/cypress/packages/server/test/unit/plugins/child/run_plugins_spec.coffee:76:26)
    at process.on (/root/cypress/packages/server/lib/plugins/child/run_plugins.js:1:1)
    at emitOne (events.js:1:1)
    at process.emit (events.js:1:1)
    at process.emit (/root/cypress/packages/ts/node_modules/source-map-support/source-map-support.js:1:1)
    at processEmit (/root/cypress/packages/server/node_modules/signal-exit/index.js:1:1)
    at processEmit [as emit] (/root/cypress/packages/launcher/node_modules/signal-exit/index.js:1:1)
    at process._fatalException (bootstrap_node.js:1:1)
```

https://circleci.com/gh/cypress-io/cypress/128235#tests/containers/1

--- a/packages/server/test/unit/plugins/child/run_plugins_spec.coffee
+++ b/packages/server/test/unit/plugins/child/run_plugins_spec.coffee
@@ -96,7 +96,7 @@ describe "lib/plugins/child/run_plugins", ->
       runPlugins(@ipc, "plugins-file")
       @ipc.on.withArgs("load").yield()
 
-      @ipc.send = (event, errorType, pluginsFile, stack) ->
+      @ipc.send = _.once (event, errorType, pluginsFile, stack) ->
         expect(event).to.eq("load:error")
         expect(errorType).to.eq("PLUGINS_FUNCTION_ERROR")
         expect(pluginsFile).to.eq("plugins-file")

--- a/packages/server/test/unit/plugins/child/run_plugins_spec.coffee
+++ b/packages/server/test/unit/plugins/child/run_plugins_spec.coffee
@@ -72,7 +72,7 @@ describe "lib/plugins/child/run_plugins", ->
       @ipc.on.withArgs("load").yields({})
       runPlugins(@ipc, "plugins-file")
 
-      @ipc.send = (event, errorType, pluginsFile, stack) ->
+      @ipc.send = _.once (event, errorType, pluginsFile, stack) ->
         expect(event).to.eq("load:error")
         expect(errorType).to.eq("PLUGINS_FUNCTION_ERROR")
         expect(pluginsFile).to.eq("plugins-file")

--- a/packages/server/test/unit/request_spec.coffee
+++ b/packages/server/test/unit/request_spec.coffee
@@ -200,6 +200,7 @@ describe "lib/request", ->
         opts = {
           url: "http://localhost:9988/econnreset"
           retryIntervals: [0, 1, 2, 3]
+          timeout: 250
         }
 
         stream = request.create(opts)
@@ -252,6 +253,7 @@ describe "lib/request", ->
         opts = {
           url: "http://localhost:9988/econnreset"
           retryIntervals: [0, 1, 2, 3]
+          timeout: 250
         }
 
         request.create(opts, true)

